packages/pages/src/dev/server/server.ts

```diff
@@ -45,6 +45,8 @@ export const createServer = async (
   app.use(finalSlashRedirect);
 
   let displayGenerateTestDataWarning = false;
+  // call generateTestData to ensure we have data to populate the index page
```

Comment:
Use caps/periods. It makes it easier to read. I originally read this and below together and had to reread to understand that they were different sentences.